jenkins/jobs/bmo_e2e_tests.pipeline

```diff
@@ -19,14 +19,38 @@ pipeline {
     stage("Run Baremetal Operator e2e tests") {
       matrix {
         agent { label "metal3-workers" }
+        // Skip redfish on PRs, test all for periodic jobs
+        when {
+          anyOf {
+            expression { env.SUSHY_TOOLS_PROTOCOL != "redfish" }
+            triggeredBy 'TimerTrigger'
+          }
+        }
         axes {
           axis {
             name 'BMO_E2E_EMULATOR'
             values 'vbmc', 'sushy-tools'
           }
+          axis {
+            name 'SUSHY_TOOLS_PROTOCOL'
+            values 'redfish', 'redfish-virtualmedia'
+          }
+        }
+        excludes {
+          exclude {
+            axis {
+              name 'BMO_E2E_EMULATOR'
+              values 'vbmc'
+            }
+            axis {
+              name 'SUSHY_TOOLS_PROTOCOL'
+              values 'redfish', 'redfish-multimedia'
```

Comment:
Is this correct? `redfish-multimedia` is not mentioned elsewhere in Metal3, and even google doesn't give results for that.

Reply:
Oh my. Thanks for spotting that!
I was expecting it not to work when I filtered out both redfish and redfish-virtualmedia. Got surprised when it worked but didn't look closer. This looks like the explanation :sweat_smile: